magento2/integration-tests/tests-integration.yml

```diff
@@ -0,0 +1,75 @@
+'🐿 - Integration Tests':
+  stage: '🦄 - Unit-/Integration Tests'
+  coverage: '/^\s*Lines:\s*\d+.\d+\%/'
+  services:
+    - name: mariadb:10.4
+      alias: db
+      command: ["--innodb-buffer-pool-size=1024M --innodb-log-buffer-size=32M --max_allowed_packet=325058560"]
+    - name: docker.elastic.co/elasticsearch/elasticsearch:7.6.2
+      alias: elasticsearch
+      command: [ "bin/elasticsearch", "-Expack.security.enabled=false", "-Ediscovery.type=single-node" ]
+  extends: .composer_cache_pull
+  needs: ["📦 - Preparation"]
+  variables:
+    PHP_BIN: ${PHP_BIN:=php}
+
+    # db config for tests-integration and artifact
+    MYSQL_ROOT_PASSWORD: root
+    APP_DB_HOST: db
+    APP_DB_USER: root
+    APP_DB_PASS: $MYSQL_ROOT_PASSWORD
+    APP_DB_NAME: db01
+
+    # elasticsearch config
+    ELASTICSEARCH_HOST: elasticsearch
+    ELASTICSEARCH_PORT: 9200
+
+    # mysql config for integration tests
+    INSTALL_CONFIG_MYSQL: >
```

Comment:
Remove this and point the Var to the default Magento path. We have to check in the `config.php` into VCS then.

Reply:
To set the ENVIRONMENT Variables, you can work with `sed`

Reply:
Replace only on lines matching the line pattern:
    `sed '/line_pattern/s/find/replace/' filename`